client/app/components/calendar/filters/RoomFilters.jsx

```diff
@@ -34,4 +37,11 @@ export default class RoomFilters extends React.Component {
   _filterEnabled(conferenceRoom) {
     return this.props.filters.find(filter => filter === conferenceRoom.id) != null;
   }
+
+  _roomCompare(leftRoom, rightRoom) {
```

Comment:
"As a user I want to have my filters always ordered in the same way so that i can predict their location"